src/main/java/org/example/algorithms/searching/FibonacciSearch.java

```diff
@@ -0,0 +1,70 @@
+package org.example.algorithms.searching;
+
+public class FibonacciSearch {
+
+    public static void main(String[] args) {
+        int[] array = {3, 5, 12, 34, 45, 67, 87, 90};
+        int search_result = search(array, 67);
```

Comment:
Rename to searchResult (camelCase)

Reply:
Made the changes, please review [PR](FibonacciSearch)